lib/matplotlib/textpath.py

```diff
@@ -39,11 +39,9 @@ def _get_font(self, prop):
     def _get_hinting_flag(self):
         return LoadFlags.NO_HINTING
 
-    def _get_char_id(self, font, ccode):
-        """
-        Return a unique id for the given font and character-code set.
-        """
-        return urllib.parse.quote(f"{font.postscript_name}-{ccode:x}")
+    def _get_glyph_id(self, font, glyph):
```

Comment:
maybe `_get_glyph_idx` or `_get_glyph_index` for consistency with above.

Reply:
Ah, this is the other name I don't want to confuse `glyph_index` with; it's a string representing the glyph, not the glyph index.

Reply:
I decided to rename this one to `glyph_repr` at least.